src/core/namespace.c

```diff
@@ -2403,6 +2403,12 @@ int setup_namespace(const NamespaceParameters *p, char **reterr_path) {
                         if (r < 0)
                                 return r;
 
+                        r = dissected_image_guess_verity_roothash(
```

Comment:
commit message:
`this doesn't not guarantee` -> `this doesn't guarantee` ?
`attestion` -> `attestation`